src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,90 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int FIRST_HALF_ARRAY = 0;
+    private static final int SECOND_HALF_ARRAY = 1;
+    private static final int SIZE_REPORT = 3;
+    private static final String ACTION_WITH_BUY = "buy";
+    private static final String ACTION_WITH_SUPPLY = "supply";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Consider renaming the method 'writeTiFile' to 'writeToFile' for better readability.